spec/requests/categories_spec.rb

```diff
@@ -0,0 +1,22 @@
+require 'rails_helper'
+
+RSpec.describe '/categories', type: :request do
+  let(:user) { create(:user) }
+  let(:category) { create(:category) }
+  before(:each) do
+    user.confirmed_at = Time.now
+    sign_in user
+  end
+
+  describe 'GET /index' do
+    it 'renders a successful response' do
+      get categories_path
+      expect(response).to render_template('categories/index')
+    end
+  end
+
+  it 'renders a successful response' do
+    get new_category_url
+    expect(response).to be_successful
+  end
```

Comment:
- [ ] When I run your tests, two examples fail. That is because you have `user.confirmed_at = Time.now`. Remember that you did not configure `devise` to use the confirmable method. I would advise you to remove this line of code in this file. 👍 

![image](https://user-images.githubusercontent.com/77106962/178076376-2b7ae024-11f5-41e6-82b9-0b8c06176826.png)

Reply:
I fixed it, I forgot to update it